src/Categories/Category/Instance/Simplex.agda

```diff
@@ -1,67 +1,152 @@
 {-# OPTIONS --without-K --safe #-}
 
-open import Categories.Category
 
+-- The Simplex category Δ
 module Categories.Category.Instance.Simplex where
 
-open import Level
+open import Level using (0ℓ)
 open import Data.Product
-open import Data.Fin.Base using (Fin; _≤_)
-open import Data.Nat.Base using (ℕ; z≤n; s≤s)
+open import Data.Fin.Base using (Fin; zero; suc; _≤_; _<_; inject₁)
+open import Data.Nat.Base using (ℕ; zero; suc; z≤n; s≤s)
 open import Function renaming (id to idF; _∘_ to _∙_)
 
-open import Relation.Binary using (_=[_]⇒_)
-open import Relation.Binary.PropositionalEquality
+open import Relation.Binary.PropositionalEquality using (_≡_; refl; sym; trans; cong)
+
+open import Categories.Category.Core using (Category)
+
+--------------------------------------------------------------------------------
+-- The Simplex Category
+--
+-- Classically, the category Δ is defined as follows:
+--   * Objects are Natural Numbers
+--   * Morphisms 'm ⇒ n' are order-preserving maps 'Fin m → Fin n'
+--
+-- We _can_ define this version of Δ, but it is generally /not/ the version we want.
+-- Most of the time we work with Δ, we only care about 2 classes of maps:
+--   * Face maps 'δᵢ : n ⇒ (1 + n)' whose image omits 'i'
+--   * Boundary maps 'σᵢ : (1 + n) ⇒ n' such that 'σᵢ i = σᵢ (i + 1) = i'
+-- It turns out that all of the maps of Δ can be decomposed into compositions
+-- of these 2 classes of maps[1], so most texts will define Functors out of Δ
+-- based solely how they act on these morphisms.
+--
+-- Therefore, we want our definition of Δ to put these morphisms front and center,
+-- by encoding our homs as follows:
+
+infix 4 _Δ⇒_
+infixr 9 _⊚_
+
+data _Δ⇒_ : ℕ → ℕ → Set where
+  ε  : ∀ {n} → n Δ⇒ n
+  δ  : ∀ {n} → (i : Fin (suc n)) → n Δ⇒ (suc n)
+  σ  : ∀ {n} → (j : Fin n) → (suc n) Δ⇒ n
+  _⊚_ : ∀ {l m n} → m Δ⇒ n → l Δ⇒ m → l Δ⇒ n
+
+-- However, this raises some tricky questions about equality of morphisms. It is tempting
+-- to use the Simplical Identities[2]:
```

Comment:
Add a comment here that you will prove these identities `face` and `degen` below?